spaceWeatherApi/DataModels/SunspotModel.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace spaceWeatherApi.DataModels
+{
+    public class SunspotData
```

Comment:
Maybe rename to `SunspotModel` to sync with the file name and other model names? (XModel format)